src/api/workspace.rs

```diff
@@ -1230,6 +1235,18 @@ async fn get_published_collab_handler(
   Ok(Json(metadata))
 }
 
+async fn get_v1_published_collab_handler(
+  path_param: web::Path<(String, String)>,
+  state: Data<AppState>,
+) -> Result<Json<AppResponse<serde_json::Value>>> {
+  let (workspace_namespace, publish_name) = path_param.into_inner();
+  let metadata = state
+    .published_collab_store
+    .get_collab_metadata(&workspace_namespace, &publish_name)
+    .await?;
+  Ok(Json(AppResponse::Ok().with_data(metadata)))
+}
```

Comment:
In this case, the response is in the form of:
```
{
   "data": {
     "ancestor_views": [...]
   }
}
```
Is this what the front ends expects? Or should it be:

{
  "data": {
    "meta": {
      "ancestor_views": [...]
    }
  }
}
```

Reply:
Yes. I have confirmed with @qinluhe on this.

Reply:
LGTM